src/theme/theme.js

```diff
@@ -0,0 +1,22 @@
+import {
+  aTextColorDarkTheme,
+  aTextColorLightTheme,
+  textColorDarkTheme,
+  textColorLightTheme,
+  toggleBorderDark,
+  toggleBorderLight
+} from '../common/cssVariables';
+
+export const lightTheme = {
```

Comment:
why not just come up with two, three, four, one...whatever number of sets of variables that have the same names:

```
const themes = {
  [LIGHT]: {
    color: 'xxx',
    backgroundColor: 'yyy',
    linksColor: 'zzz',
    someOtherRepeatedStuffColor: 'bbb'
  },
  [DARK]: {
    color: 'ggg',
    backgroundColor: 'lll',
    linksColor: 'mmm',
    someOtherRepeatedStuffColor: 'ooo'
  },
  [SHITTY]: {
    color: 'vvv',
    backgroundColor: 'hhh',
    linksColor: 'ggg',
    someOtherRepeatedStuffColor: 'iii'
  }
}
```

and then just set current `theme[LIGHT | DARK | SHITTY | DEFAULT]` onto the state when toggle. 

Everywhere in your app you just read `theme.color`, `theme.someOtherRepeatedStuffColor`, `theme.linksColor` etc